Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -262,6 +263,7 @@ struct PerWebProcessState {
     NSUInteger _partialIntelligenceTextAnimationCount;
     BOOL _writingToolsTextReplacementsFinished;
 #endif
+    RetainPtr<STWebpageController> _screenTimeWebpageController;
```

Comment:
Guarding?

Reply:
I think you could also consider just keeping track of a single `RetainPtr<STWebpageController>` ivar/member variable, which means deleting either this ivar (and using `_page->screenTimeWebpageController()`) or deleting `m_screenTimeWebpageController` on `WebPageProxy` (and instead adding a `PageClient` hook that asks the web view for the controller).